pkg/config/constants.go

```diff
@@ -117,6 +117,8 @@ const (
 	// SparkDriverServiceAccountName is the Spark configuration key for specifying name of the Kubernetes service
 	// account used by the driver pod.
 	SparkDriverServiceAccountName = "spark.kubernetes.authenticate.driver.serviceAccountName"
+	// account used by the executor pod.
+	SparkExecutorAccountName = "spark.kubernetes.authenticate.executor.serviceAccountName"
```

Comment:
Is this config option available in Spark on Kubernetes?

Reply:
Seems so 

https://github.com/apache/spark/blob/79ea014c68d0930e1b9ac313ef4edaaec1e63882/resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala#L103

https://github.com/apache/spark/blob/79ea014c68d0930e1b9ac313ef4edaaec1e63882/resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala#L150

Reply:
OK, looks like the doc was not updated.